src/components/KeyboardToolbar/index.tsx

```diff
@@ -146,7 +150,7 @@ const KeyboardToolbar: React.FC<KeyboardToolbarProps> = ({
 
   return (
     <KeyboardStickyView offset={offset}>
-      <View style={toolbarStyle} testID={TEST_ID_KEYBOARD_TOOLBAR}>
+      <View style={toolbarStyle} testID={TEST_ID_KEYBOARD_TOOLBAR} {...rest}>
```

Comment:
didn't work if I spreaded the props to the KeyboardStickyView. Since the underlying component is a Reanimated view, I suspect it kind of messes with onLayout events

Reply:
Yeah @CorentinAndre 

In my experiments it always reported `height` as `0` 🤷‍♂️